db/models.py

```diff
@@ -50,3 +61,46 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(default=now)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+
+    def __str__(self) -> str:
+        return f"{self.created_at.strftime('%Y-%m-%d %H:%M:%S')}"
```

Comment:
The `__str__` method in the `Order` model could be improved by ensuring consistent string formatting. Consider using single quotes for the outer string or escaping the inner double quotes to avoid potential syntax errors.